data/data/baremetal/main.tf

```diff
@@ -1,44 +1,31 @@
-provider "libvirt" {
-  uri = "${var.libvirt_uri}"
-}
+resource "ironic_node_v1" "openshift-master" {
```

Comment:
I think this belongs in baremetal/masters, the base main.tf sets up the providers and modules

Reply:
There was a wrong file copy there